### PR TITLE
Issue #13345: Enable example test for parameterassignment

### PR DESCRIPTION
**Pull request by AayushSaini101**
Related: https://github.com/checkstyle/checkstyle/issues/13345

### CODE REVIEW

**Review by nrmancuso — APPROVED** [review]

File: src/xdocs/checks/coding/parameterassignment.xml [hunk 1]
```diff
@@ -53,13 +53,16 @@ class MyClass {
...
   IntPredicate obj = a -&gt; ++a == 12; // violation
   IntBinaryOperator obj2 = (int a, int b) -&gt; {
-      a++;     // violation
-      b += 12; // violation
-      return a + b;
+
+    a++;  //violation
+    b += 12; //violation
+    return a+b;
+
```
Comment (rnveach) [inline src/xdocs/checks/coding/parameterassignment.xml:60]: Remove extra blank lines that you added.

File: src/xdocs/checks/coding/parameterassignment.xml [hunk 2]
```diff
@@ -53,13 +53,16 @@ class MyClass {
...
   };
   IntPredicate obj3 = a -&gt; {
-      int b = a; // ok
-      return ++b == 12;
+
+    int b = a; // ok
```
Comment (rnveach) [inline src/xdocs/checks/coding/parameterassignment.xml:64]: same.

**Review by rnveach — CHANGES_REQUESTED** [review]

**Comment by rnveach** [comment]
@AayushSaini101 Please also keep the number of commits in this PR as 1.

**Comment by AayushSaini101** [comment]
> @AayushSaini101 Please also keep the number of commits in this PR as 1.

Working on the fix

**Comment by AayushSaini101** [comment]
> @AayushSaini101 Please also keep the number of commits in this PR as 1.

Done

**Comment by romani** [comment]
@AayushSaini101 , please fix CI

**Comment by AayushSaini101** [comment]
> @AayushSaini101 , please fix CI

Tried. Some failing reason due to the switching between the acccounts

**Comment by romani** [comment]
Prage is failing, please pull latest master from us and rebase.

**Review by rnveach — CHANGES_REQUESTED** [review]
There is some issue with the PR with 133 changed files. We cannot merge until this is only showing your changes in GitHub.

In addition:
> The PR is based on a master that is 21 commit(s) old.
The max allowed is 10.

**Comment by AayushSaini101** [comment]
> There is some issue with the PR with 133 changed files. We cannot merge until this is only showing your changes in GitHub.
> 
> In addition:
> 
> > The PR is based on a master that is 21 commit(s) old.
> > The max allowed is 10.
> 
> https://github.com/checkstyle/checkstyle/wiki/PR-rules See 2 as that may help.

Can I open new PR I guess it would be better as I switched to single user

**Comment by rnveach** [comment]
If you can't re-push to the original branch, then be welcome opening a new PR. Just ping us here on the new one.

**Comment by AayushSaini101** [comment]
> If you can't re-push to the original branch, then be welcome to open a new PR. Just ping us here on the new one.

Closing this PR Opening a new

**Pull request closed without merging**